vignettes/snomedizer.Rmd

```diff
@@ -0,0 +1,373 @@
+---
+title: "snomedizer: R Interface to the SNOMED CT Terminology Server REST API"
+date: '`r format(Sys.Date(), "%d %B %Y")`'
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    always_allow_html: yes
+    df_print: tibble
+vignette: >
+  %\VignetteIndexEntry{snomedizer: R Interface to the SNOMED CT Terminology Server REST API}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(dplyr)
+library(snomedizer)
+```
+
+# Overview
+
+## What is SNOMED CT?
+
+SNOMED CT is an international ontology of clinical terms. You can think of it as an encyclopedia on all things that are talked about in medicine, for instance: anatomical structures, diagnoses, medications, tests,  surgical procedures, pathogens such as SARS-CoV-2 or *Candidida auris*, or even clinical findings, such as a high temperature.
+
+SNOMED CT is used mainly in the UK:
+
+* to record information about patient care in general practice and hospitals
+* to analyse electronic health records, in particular medical notes.
+
+Increasingly, other countries are contributing towards and using SNOMED CT.
+
+The best ways to learn more about SNOMED CT are:
+
+* the [SNOMED International Browser](https://browser.ihtsdotools.org/)
+* the [SNOMED CT for Data Analysts](https://elearning.ihtsdotools.org/enrol/index.php?id=26) free online learning course
+* the official [SNOMED CT Starter Guide](http://snomed.org/sg), particularly chapters [5. Logical Model](http://snomed.org/sg/5.+SNOMED+CT+Logical+Model) and [6. Concept Model](http://snomed.org/sg/6.+SNOMED+CT+Concept+Model).
+
+## What is snomedizer?
+
+`snomedizer` is an R package to query and manipulate the SNOMED CT terminology.
+
+It allows you to perform the same operations as the [SNOMED International Browser](https://browser.ihtsdotools.org/).
+
+`snomedizer` is based on [Snowstorm](https://github.com/IHTSDO/snowstorm/), the official SNOMED CT Terminology Server. Snowstorm is the back-end terminology service for both the [SNOMED International Browser](https://browser.ihtsdotools.org/) and the SNOMED International Authoring Platform.
+
+# Introduction to SNOMED CT
+
+SNOMED CT contains rich knowledge about medical concepts.
+
+Try for yourself with the quick exercise below:
+
+1. Go to the [SNOMED CT International Browser](https://browser.ihtsdotools.org/).
```

Comment:
Add a note that one may be prompted to accept license terms if one hasn't done so before